doc/api/pymongo/event_loggers.rst

```diff
@@ -0,0 +1,6 @@
+:mod:`event_loggers` -- Example loggers
```

Comment:
This file isn't actually included anywhere in the API docs so it can't be navigated to. You need to add this file to the toctree in `doc/api/pymongo/index.rst` for it to show up in the appropriate location.

Reply:
Done.